projects/fab-speed-dial/src/lib/fab-speed-dial.ts

```diff
@@ -111,12 +113,13 @@ export class EcoFabSpeedDialActionsComponent implements AfterContentInit {
     styleUrls: ['fab-speed-dial.scss'],
     encapsulation: ViewEncapsulation.None,
 })
-export class EcoFabSpeedDialComponent implements AfterContentInit {
+export class EcoFabSpeedDialComponent implements OnDestroy, AfterContentInit {
     private isInitialized = false;
     private _direction: Direction = 'up';
     private _open = false;
     private _animationMode: AnimationMode = 'fling';
-
+    private _closeOnOutsideClick = false;
+    private _documentListener: () => void;
```

Comment:
This a function or null:

```suggestion
    private _documentListener: () => void | null = null;
```